test/util/testnode/genesis_options.go

```diff
@@ -34,7 +34,7 @@ func SetBlobParams(codec codec.Codec, params blobtypes.Params) GenesisOption {
 func ImmediateProposals(codec codec.Codec) GenesisOption {
 	return func(state map[string]json.RawMessage) map[string]json.RawMessage {
 		gs := v1.DefaultGenesisState()
-		gs.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1)))
+		gs.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(10)))
```

Comment:
since this purpose of this function is to make proposals fast, I don't think we need to change this as well do we?

Reply:
From my understanding as we are changing the markdown file, it also makes sense to change that in the files which are having the impact.